statictemplate/management/commands/statictemplate.py

```diff
@@ -82,10 +82,15 @@ class Command(BaseCommand):
                     action='store',
                     dest='output',
                     help='Output file'),
+        make_option('--language-code', '-l',
+                    action='store',
+                    dest='language_code',
+                    help='Language Code')
     )
 
     def handle(self, template, language=None, extra_request=None, **options):
         request = {}
+        language = options.get('language_code',False)
```

Comment:
Fix style here (``get('language_code', False)`` not ``get('language_code',False)``